src/plugins/wizard/public/visualizations/metric/to_expression.ts

```diff
@@ -171,5 +171,5 @@ export const toExpression = async ({ style: styleState, visualization }: MetricR
 
   const ast = buildExpression([opensearchaggs, metricVis]);
 
-  return ast.toString();
+  return `opensearchDashboards | opensearch_dashboards_context | ${ast.toString()}`;
```

Comment:
I assume this change is intended, and maybe it's obvious for someone who understands expressions, but an explanatory comment would seem useful.

Reply:
The `toExpression` function needs some TLC as a whole. I plan on refactoring this when we have more than one viz type. `opensearchDashboards | opensearch_dashboards_context` are the expression functions that get the top nav context and pass it along to the agg function. For now ill leave it as is since this is how the `build_pipeline.ts` does it too. 

I want to abstract away most of this logic into utility functions so that all we have here is some sort of builder function that simply chains the expression function builders.